docs/source/conf.py

```diff
@@ -519,6 +519,7 @@
     "is_initialized",
     "is_mpi_available",
     "is_nccl_available",
+    "is_xccl_available",
```

Comment:
Adding new entries to this file is not ok.
Given you added the function to the .rst below, this should not be needed unless something is wrong with the doc setup there.

Reply:
@zhangxiaoli73, please add the full name `torch.distributed.distributed_c10d.is_xccl_available` to the doc and remove `is_xccl_available` from this list.

Reply:
@albanD Thanks for your reminder. Revoked the change in this file. Please help review.